src/test/suite/completion.java.test.ts

```diff
@@ -19,22 +19,23 @@
 import * as vscode from 'vscode';
 import { areJavaDependenciesDownloaded } from '../../JavaDependenciesManager';
 import { getDocUri, checkExpectedCompletion } from './completion.util';
+import * as os from 'os';
+import waitUntil = require('async-wait-until');
+import { test, describe} from 'mocha';
 
-const os = require('os');
-const waitUntil = require('async-wait-until');
-
-suite('Should do completion in Camel K standalone files', () => {
+describe('Should do completion in Camel K standalone files', () => {
 
 	const docUriJava = getDocUri('MyRouteBuilder.java');
-
 	const expectedCompletion = { label: 'from(String uri) : RouteDefinition'};
 
-	var testVar = test('Completes from method for Java', async () => {
-		if(os.homedir().includes('hudson')) {
-			testVar.skip();
-		}
-		await testCompletion(docUriJava, new vscode.Position(5, 11), expectedCompletion);
-	}).timeout(76000);
+	test('java completion method', function() {
```

Comment:
writing it this way is no more checking anything. The test is always passing in 0ms, it doesn't wait for the check to happen

Reply:
Reverted change